src/csharp/Grpc.Core/Internal/NativeLogRedirector.cs

```diff
@@ -51,38 +52,41 @@ public static void Redirect(NativeMethods native)
             }
         }
 
+        [AOT.MonoPInvokeCallback(typeof(GprLogDelegate))]
         private static void HandleWrite(IntPtr fileStringPtr, int line, ulong threadId, IntPtr severityStringPtr, IntPtr msgPtr)
         {
             try
             {
-                var logger = GrpcEnvironment.Logger;
+                /* var logger = GrpcEnvironment.Logger;*/
                 string severityString = Marshal.PtrToStringAnsi(severityStringPtr);
                 string message = string.Format("{0} {1}:{2}: {3}",
                     threadId,
-                    Marshal.PtrToStringAnsi(fileStringPtr), 
-                    line, 
+                    Marshal.PtrToStringAnsi(fileStringPtr),
+                    line,
                     Marshal.PtrToStringAnsi(msgPtr));
-                
+
                 switch (severityString)
                 {
                     case "D":
-                        logger.Debug(message);
+                        // logger.Debug(message);
+                        Logz.LogDebug(message, "GRPC");
```

Comment:
This is my own logging tool, which I'd rather route everything through anyways. I suppose I could have conformed to the `ILogger` interface and assigned it to the `GrpcEnvironment.Logger` instead.